js/iframe-coordinator/src/messages/NavRequest.ts

```diff
@@ -5,6 +5,6 @@ export interface NavRequest {
 }
 
 export interface LabeledNavRequest extends LabeledMsg {
-  msgType: 'navRequest';
+  msgType: string;
```

Comment:
If this is ok with you, we can change the types around a bit to only have the | on the msg itself.  So ClientToHost can be msg: NavRequest | Toast | Publication